### PR TITLE
fixed localhost response 

### PR DESCRIPTION
**Pull request by sujitmohanty**
1. Composed & defined a variable named `baseUrl` inside `config > development.js`
2. Assigned it to the concerned keys inside of rdsApi & rdsUi object literals.
3. Updated README.md
4. Immaculated code wherever necessary by removing all the console logs statements.

### CODE REVIEW

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]
@sujitmohanty , seems like there are too many changes in this PR, most of which are around indentation. Can you remove them?
Also, seems like you are trying to redirect the login to localhost. This should not be environment specific.
Related issue: https://github.com/Real-Dev-Squad/website-backend/issues/62

**Comment by sujitmohanty** [comment]
Ok Ankur.

File: controllers/authController.js [hunk 1]
```diff
@@ -13,38 +13,44 @@ const authService = require('../services/authService')
  */
 const githubAuth = (req, res, next) => {
   let userData
-  const authRedirectionUrl = `${config.get('services.rdsUi.baseUrl')}${config.get('services.rdsUi.routes.authRedirection')}`
+  const authRedirectionUrl = `${config.get(
```
Comment (ankurnarkhede) [inline controllers/authController.js:16]: Can you remove all the unwanted code formatting from the PR?
Please submit PR with changes only in the code ehich id to be modified.
Seems your local code formatter is messing the code!

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]

**Comment by sujitmohanty** [comment]
I had removed all the "unwanted" code formatting on Oct 14 itself, after I
was intimated about them. If they still persist in a specific file/files,
please let me know. I have had removed them meticulously and am unable to
discover anything in usersController.js

**Comment by Rucha1499** [comment]
Fixed this issue in #127 . 
Therefore, closing this one.

**Pull request closed without merging**